### DISCUSSION TITLE
Welcome to yardim-agi-flutter Discussions!

### DISCUSSION THREAD

**Discussion by BRTZL** [acikkaynak/yardim-agi-flutter#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lindagarmel11** [comment]
> 
> 
> ## 👋 Welcome!
> 
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>   * Ask questions you’re wondering about.
> 
>   * Share ideas.
> 
>   * Engage with other community members.
> 
>   * Welcome others and are open-minded. Remember that this is a community we
> 
>   build together 💪.
> 
> 
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
> 
> <!--
> 
>   For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.
> 
> 
> 
>   📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.
> 
> 
> 
>   🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).
> 
> 
> 
>   ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.
